content/docs/resiliency/deployment.md

```diff
@@ -119,6 +121,34 @@ podmon:
 
 ```
 
+## PowerScale Specific Recommendations
+
+Here is a typical installation used for testing:
+
+```yaml
+podmon:
+  image: dellemc/podmon
+  enabled: true
+  controller:
+    args:
+      - "-csisock=unix:/var/run/csi/csi.sock"
+      - "--labelvalue=csi-isilon"
+      - "--arrayConnectivityPollRate=60"
+      - "--driverPath=csi-isilon.dellemc.com"
+      - "--mode=controller"
+      - "--skipArrayConnectionValidation=false"
+      - "--driver-config-params=/csi-isilon-config-params/driver-config-params.yaml"
```

Comment:
There is a new label for csi driver pod, see example in csi unity or csi powerflex code

Reply:
Added to all sections in this doc.